visual-dotnet/SauceLabs.Visual/VisualClient.cs

```diff
@@ -46,6 +65,11 @@ public VisualClient(WebDriver wd, Region region, string username, string accessK
         /// <param name="buildOptions">the options of the build creation</param>
         public VisualClient(WebDriver wd, Region region, string username, string accessKey, CreateBuildOptions buildOptions)
         {
+            if (string.IsNullOrWhiteSpace(username) || string.IsNullOrEmpty(accessKey))
```

Comment:
Should these two `IsNull...` checks be the same?

Reply:
They definitively should !